src/ert/storage/local_ensemble.py

```diff
@@ -262,39 +227,30 @@ def _load_single_dataset(
     def _load_dataset(
         self,
         group: str,
-        realizations: Union[int, Sequence[int]],
+        realizations: Union[int, Sequence[int], None],
     ) -> xr.Dataset:
         if isinstance(realizations, int):
-            return self._load_single_dataset(group, realizations)
+            return self._load_single_dataset(group, realizations).isel(
+                realizations=0, drop=True
```

Comment:
`isel` is indexed select. `isel(realizations=0)` selects the first realization, not necessarily the realization with value 0. Which we know exists because we create all files with this realization coordinate.

Although we should probably swap it around now that we're not using `open_mfdataset`.